picobot/handlers.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import dataset
+from telegram import Bot, Update, Message
+
+from functools import wraps
+
+from .config import CREATOR_ID, DB_PATH, ROOT_DIR
+from picobot import responses
+from .msg_type import MsgType
+from .painter import sticker_from_text, sticker_from_image
+
+IMG_DIR = ROOT_DIR + '/images/'
+IMG_NAME = 'img'
+AVATAR_NAME = 'avatar'
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
+)
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_EMOJI = '😁'
+
+
+def creator_only(func):
+    @wraps(func)
+    def new_func(bot, update, *args, **kwargs):
+        if update.message.from_user.id == CREATOR_ID:
+            return func(bot, update, *args, **kwargs)
+        else:
+            update.message.reply_text(responses.ACCESS_DENIED)
+
+    return new_func
+
+
+def text_handler(bot, update):
+    pass
+
+
+def start(bot, update):
+    """Send a message when the command /start is issued."""
+    update.message.reply_text(responses.GREETING)
+
+
+# def new_pack(bot: Bot, update: Update):
+#     db = dataset.connect(f'sqlite:///{DB_PATH}')
+#     table = db.get_table('users', primary_id='id')
+#     user_id = update.message.from_user.id
+#     user = table.find_one(id=user_id)
+#     if not user:
+#         table.insert(id=user_id)
+
+#     pack_number = create_pack()
+#     if pack_number == 0:
+#         update.message.reply_text(responses.ERROR_MSG)
+#     else:
+#         update.message.reply_text(f'{responses.PACK_CREATED}: {pack_number}')
+
+
+def create_pack(bot: Bot, update: Update):
+    user_id = update.message.from_user.id
+    if not check_msg_format(update.message.text):
+        update.message.reply_text(responses.INVALID_MSG)
+    splittext = update.message.text.split()
+    title = splittext[1]
+    name = title + '_by_' + bot.username
+    png_sticker = open(f'{IMG_DIR}caravela.png', 'rb')
```

Comment:
Use context managers:

```python
with open(f'{IMG_DIR}caravela.png', 'rb') as png_sticker:
    ...
```

And then remove `png_sticker.close()` (context-manager will handle it properly).